.github/ISSUE_TEMPLATE/reportstream-team-member-offboarding.md

```diff
@@ -18,9 +21,13 @@ Note: please do not put sensitive information here (issues are public)
 - [ ] Docker
```

Comment:
maybe make this `Docker Desktop` instead to remove ambiguity?

Reply:
Oh, good point. Do you think it's better as `Docker Desktop license assignment`?

Reply:
Yep, that's even better!